packages/ui/src/common/components/buttons/Toggle.tsx

```diff
@@ -12,10 +12,12 @@ interface ToggleableItemProps {
   children: ReactNode | ((isOpen: boolean) => ReactNode)
   absoluteToggle?: boolean
   className?: string
+  isOpen: boolean
+  toggleOpen: () => void
 }
 
-export const ToggleableItem = ({ children, absoluteToggle, className }: ToggleableItemProps) => {
-  const [isOpen, toggleOpen] = useToggle()
+export const ToggleableItem = ({ children, absoluteToggle, className, isOpen, toggleOpen }: ToggleableItemProps) => {
+  // const [isOpen, toggleOpen] = useToggle()
 
   return (
     <Toggle isOpen={isOpen} absoluteToggle={absoluteToggle} className={className}>
```

Comment:
Thanks for the PR. It works but it introduces some necessary changes. While this:
```suggestion
    <Toggle isOpen={isOpen} absoluteToggle={absoluteToggle} className={className} onClick={toggleOpen}>
```
And removing the `onClick` from `ToggleButton` should work too without changing the component behavior.

Also please try to not leave commented out code in the PRs

Reply:
Please check again, it works without changing the component behavior
Removed all comments that had left there before

Reply:
By "component behavior" I meant the behavior of `ToggleableItem`, in the current code base `ToggleableItem` manages the state of the toggle (which IMO makes it a bit nicer to use), while in this PR the parent component keeps the state. This leads to the **unnecessary* changes I was talking about (sorry about the typo).

It's not too bad because it seems `ToggleableItem` is currently only used in `MemberRoleToggle`, but please try to keep the changes minimum for smaller this PRs like this one.

Reply:
@ytdbuolf I'm sorry for the very late reply.

The icon you are referring to is the arrow, right ? In this case, yes both the panel and icon should be clickable.

But the way I see it, since the icon is within the `onClick` should be on the panel (i.e `<Toggle />`) and not on the icon (i.e `<ToggleButton><Arrow/><ToggleButton>`). Is this what are proposing in your latest comment ? If so, is there a drawback to this solution that I am missing ?